### ISSUE TITLE
Conflicting extruder specific values For the build plate temperature are not correctly displayed.

### ISSUE THREAD

**Issue by plaintoothpaste** [Ultimaker/Cura#12738]
### Application Version

4.13.1

### Platform

Windows 10

### Printer

Ultimaker s5

### Reproduction steps

1. Select a first material (eg PLA)
2. Select a second material with a different built platform temperature (eg PETG)
3. The `Build plate temperature` is shared between these two materials and the maximum value is always used. However, The info popup (shown when you hover over the link icon), only shows the second material value two times instead of showing the two different values. 

### Actual results

![context menu_1](https://user-images.githubusercontent.com/31958211/178273921-e74032e0-ddac-4b3b-8e75-c1c583b3ebcc.PNG)

### Expected results

![context menu_2](https://user-images.githubusercontent.com/31958211/178274594-780c392c-84f3-44ac-bb15-82e0723a257c.PNG)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

### Additional information & file uploads

This is also an issue for  `build plate temperature initial layer` and potentially other settings that can conflict between extruders. However, for `Build volume temperature` this works correctly.

**Comment by Ghostkeeper** [comment]
Yes, I can reproduce this issue. Indeed it's expected to show the unresolved values, not the resolved value twice.

I'll raise this with the team to see if we can fix it soon-ish.

**Comment by Ghostkeeper** [comment]
We can't, apparently. You had a good eye. But this is a small detail, and we have bigger fish to fry.

We'll keep this issue open but it's not planned in to fix yet.
